app/boards/shields/jian/jian.zmk.yml

```diff
@@ -0,0 +1,12 @@
+file_format: "1"
+id: jian
+name: Jian
+type: shield
+url: https://github.com/KGOH/Jian-Info
+requires: [pro_micro]
+features:
+  - keys
+  - underglow
```

Comment:
The Jian doesn't feature underglow, only single-color backlighting (which, AFAIK, isn't supported yet).

Reply:
Thanks. Fixed.